sources/packages/web/src/services/EducationProgramService.ts

```diff
@@ -183,4 +184,16 @@ export class EducationProgramService {
       payload,
     );
   }
+
+  /**
+   * Allows a program to be deactivated.
+   * @param programId program to be deactivated.
+   * @param payload information to support the deactivation.
+   */
+  async deactivateProgram(
+    programId: number,
+    payload?: DeactivateProgramAPIInDTO,
```

Comment:
Is there a reason to not prefer options here??

Reply:
No extra properties needed or planned so I kept it as it is right now.
Is the idea that any optional property will always be present inside an options parameter?

Reply:
Not necessarily, keeping it inside options make it easy to extend the optional properties as we know. Im good here.